src/couch_log/test/couch_log_formatter_test.erl

```diff
@@ -45,7 +45,7 @@ gen_server_error_test() ->
         {
             Pid,
             "** Generic server and some stuff",
-            [a_gen_server, {foo, bar}, server_state, some_reason]
+            [a_gen_server, {foo, bar}, server_state, some_reason, sad, args]
```

Comment:
You should add a new test case so that we're testing with and without the extra args.